src/components/visx/XYChartTooltipWithBounds.tsx

```diff
@@ -281,6 +287,7 @@ const TooltipInner = <Datum extends object>({
       )}
 
       {glyphProps.map(({ x, y, ...props }, i) => (
+        // eslint-disable-next-line react/no-array-index-key
```

Comment:
[suggestion] probably worth adding the `x,y` coords to the key or something. i find index keys actually cause issues often enough where i avoid suppressing this error.

Reply:
oh I think this is all unnecessary - glyphs already have a key so we can just kill the fragment.